notebooks/data-sources/TestGrid/metrics/number_of_flakes.ipynb

```diff
@@ -18,7 +18,7 @@
   },
```

Comment:
<p>Here it might be better to show a histogram of severity scores. And then output the top 5 tests with the highest flake score. Then add a short paragraph about the results.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Sure.</p><p> </p>

Reply:
<p>Can you add another graph below that zooms in on the very bottom of the histogram so we can see the distribution of severity? And add a comment like "We can see here that we've gone from about 8,000 tests to investigate for probable flakiness to less then 100. We can also clearly order them by severity" or something like that? I'm making this suggestion because the plot above looks like 100% of the tests have severity 0, which we know is not the case. : ) </p><pre class="ql-syntax" spellcheck="false">plt.figure(fig,size = ((15,10)))
flake_severity.hist(bins=50)
plt.ylabel("Tests")
plt.xlabel("Flake Severity")
plt.ylim((0,25))
plt.show()
</pre>

Reply:
That makes sense. I will add it :+1: